### PR TITLE
Display Qt5 version in testsuite result pages

### PR DESCRIPTION
**Pull request by lrineau**

### CODE REVIEW

File: Maintenance/test_handling/create_testresult_page [hunk 1]
```diff
@@ -450,9 +450,7 @@ sub print_platform_descriptions()
 <th>BOOST</th>
 <th>MPFR</th>
 <th>GMP</th>
-<th>QT3</th>
-<th>QT4</th>
-<th>OPENGL</th>
+<th>QT5</th>
```
Comment (sloriot) [inline Maintenance/test_handling/create_testresult_page]: Are you sure you want to skip Qt3 here?

Comment (lrineau) [inline Maintenance/test_handling/create_testresult_page]: That line is about the display, in the test results, of the Qt3 version. Nobody cares of the precise Qt3 version that is installed. Either Qt3 is installed and configured in CMake, and in that case that is probably Qt-3.8.x, or it is not installed or configured. We can see the absence of Qt3 by the blue cells in the corresponding `*_Demo` lines.

I can restore the display of the Qt3 version, but I do not think that is necessary.

**Comment by lrineau** [comment]
It is tested, successfully: the test results pages now display the version of Qt5, and Qt3 and OpenGL versions are no longer displayed.

https://cgal.geometryfactory.com/CGAL/Members/testsuite/results-4.7-Ic-106.shtml

**Comment by sloriot** [comment]
@lrineau  You have an issue with the number of columns and the alignement associated in the first table.

**Comment by lrineau** [comment]
Indeed.

**Comment by lrineau** [comment]
I filled #244 to remember that there is a bug.

**Pull request merged**